### PR TITLE
DOCS-2276: Adds limitation for logging on OSS Windows

### PR DESCRIPTION
**Pull request by ctauchen**
Direct preview: https://deploy-preview-1589--tigera.netlify.app/calico/latest/getting-started/kubernetes/windows-calico/limitations

### CODE REVIEW

**Comment by ctauchen** [comment]
@fasaxc

This adds the same row we have in both CE and CC. Can only presume it was missed at some point.

File: calico/getting-started/kubernetes/windows-calico/limitations.mdx [hunk 1]
```diff
@@ -12,6 +12,7 @@ description: Review limitations before starting installation.
...
 | Security            | **Not supported**: <br />- Application Layer Policy (ALP) for Istio<br />- Policy for hosts (host endpoints, including automatic host endpoints)<br />- Encryption with WireGuard                                                                                                                                                                                                                                                                                                            |
+| Visibility and troubleshooting | **Not supported**:<br/>- Packet capture<br />- DNS logs<br />- iptable logs<br />- L7 logs                                                                                                                                                                                      |
```
Comment (fasaxc) [inline calico/getting-started/kubernetes/windows-calico/limitations.mdx]: OS doesn't have DNS logging/L7 logging in general so Windows is no exception.

I'd express "iptables logs" as "'Log' action in network policy"

**Review by fasaxc — CHANGES_REQUESTED** [review]

File: calico/getting-started/kubernetes/windows-calico/limitations.mdx (hunk 1 above)
Comment (ctauchen) [reply calico/getting-started/kubernetes/windows-calico/limitations.mdx]: Thanks, changes made.

File: calico/getting-started/kubernetes/windows-calico/limitations.mdx [hunk 2]
```diff
@@ -12,6 +12,7 @@ description: Review limitations before starting installation.
...
 | Security            | **Not supported**: <br />- Application Layer Policy (ALP) for Istio<br />- Policy for hosts (host endpoints, including automatic host endpoints)<br />- Encryption with WireGuard                                                                                                                                                                                                                                                                                                            |
+| Visibility and troubleshooting | **Not supported**:<br/>- Packet captured<br />- 'Log' action in network policy                                                                                                                                                                                      |
```
Comment (fasaxc) [inline calico/getting-started/kubernetes/windows-calico/limitations.mdx]:
```suggestion
| Visibility and troubleshooting | **Not supported**:<br />- 'Log' action in network policy                                                                                                                                                                                      |
```

Comment (ctauchen) [reply calico/getting-started/kubernetes/windows-calico/limitations.mdx]: Done.

**Review by fasaxc — APPROVED** [review]

**Pull request merged**